R/pkg/inst/tests/testthat/test_sparkSQL.R

```diff
@@ -1137,6 +1137,13 @@ test_that("string operators", {
   expect_equal(count(where(df, like(df$name, "A%"))), 1)
   expect_equal(count(where(df, startsWith(df$name, "A"))), 1)
   expect_equal(first(select(df, substr(df$name, 1, 2)))[[1]], "Mi")
+  if (as.numeric(R.version$major) >= 3 && as.numeric(R.version$minor) >= 3) {
```

Comment:
Do you know why this is needed? Jenkins is running with R 3.2~, not R 3.3.0, so this check has effectively disabled all the tests below.